### PR TITLE
Batch modify v5.1 docs

### PR DESCRIPTION
**Pull request by Joyinqin**
Signed-off-by: Joyinqin <joyinqin@pku.edu.cn>

### What is changed?

- content/docs/5.1/develop/clients/go.md
- content/docs/5.1/develop/clients/introduction.md
- content/docs/5.1/develop/rawkv/introduction.md
- content/docs/5.1/develop/rawkv/scan.md
- content/docs/5.1/reference/architecture/introduction.md
- content/docs/5.1/reference/introduction.md
- content/docs/5.1/reference/architecture/terminology.md

### Any related PRs or issues?
#243 #236 #238 #240 #239

### CODE REVIEW

**Review by marsishandsome — APPROVED** [review]
LGTM

File: content/docs/5.1/reference/architecture/introduction.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
         weight: 1
 ---
 
-This section introduces how TiKV works and how it was built:
+This chapter introduces how TiKV works and how it was built:
 
-- [Overview](../overview):  The overview of TiKV architecture.
-- [Storage](../storage): Understand the storage layer of a TiKV cluster.
+- [Overview](../overview): The overview of TiKV architecture.
+- [Storage](../storage): Understands the storage layer of a TiKV cluster.
```
Comment (qiancai) [inline content/docs/5.1/reference/architecture/introduction.md:13]:
```suggestion
- [Storage](../storage): Introduces the storage layer of a TiKV cluster.
```

File: content/docs/5.1/reference/architecture/terminology.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 
+This document explains some important terminologies of TiKV and PD.
+
 ## Node
 
-A TiKV **Node** is a physical node in the cluster, which could be a virtual machine, a container, etc. Within each Node, there can be one or more stores.
+A TiKV **Node** is a physical node in the cluster, which might be a virtual machine, a container, etc. Within each Node, there can be one or more stores.
 
-The status for the Node, Store, and Region will be regularly reported to the Placement Driver.
+The status for the Node, Store, and Region is regularly reported to the Placement Driver.
```
Comment (qiancai) [inline content/docs/5.1/reference/architecture/terminology.md:16]:
```suggestion
The Node, Store, and Region regularly report their status to the Placement Driver.
```

**Review by qiancai — COMMENTED** [review]
@Joyinqin The review is done. Please check the comments. Thanks!

**Comment by Joyinqin** [comment]
@qiancai PTAL again,thx~

**Review by qiancai — APPROVED** [review]

**Pull request merged**